agent-skills/README.md

```
@@ -0,0 +1,70 @@
+
+### Method lookups
+
+For live method and type lookups against the SDK while building, run the docs MCP server:
+
+```bash
+npx -y @yellow-org/sdk-mcp@^1
+```
+
+## Skill structure
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)